test/spec/spec.model.js

```diff
@@ -240,6 +241,22 @@ describe('Model model', function () {
             model.url.should.have.been.calledWithExactly({ url: 'foo' });
         });
 
+        it('can handle a parsed URL object', function () {
```

Comment:
Should there be equivalent tests for fetch and delete? Or perhaps this test isn't required? (Could "allows custom headers" be adapted to test as much as this tests?)